CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.36.2
+* Cleanup the gemspec. No code changes.
```

Comment:
probably then we dont need to release a new version.

Reply:
that was my question in the PR description, the post I linked to states

>　We understand that there are times when you need to get things done. If you are genuinely in a situation where you can’t release a new version to update those links, please feel free to file a support ticket and for the time being, we will update it for you. Having said that, we also hope that you feel metadata links are a useful addition and you will use them in your next versions release.

...which infers that releasing a new version is needed but I'm unsure if new version _requires_ a number bump or if you can re-release.